Kempe/Kempe(Linear).java

```diff
@@ -0,0 +1,125 @@
+//This code is contributed by Riyazul555
+
+// Time Complexity = O(N^2 + N * (N+M))
```

Comment:
We're working on dense graphs, so this can be simplified to O(N*(N+M)), since N*(N+M) > N² anyway.